src/components/ProgramList.js

```diff
@@ -170,24 +168,6 @@ class ProgramList extends Component {
           ) : (null)
         }
         <Grid container direction="column" justify="center" alignItems="center">
-          <Grid item container direction="row" justify="flex-end">
-            <Grid item>
-              <NewButton
```

Comment:
I moved this button down a level so that it could be in line with the collection title, as shown in the mock.